packages/frontend/src/PortalForm/index.tsx

```diff
@@ -70,10 +71,20 @@ const getError = (name: string, errors: InputError[]): string | null => {
 
 const MappingBar = () => {
   const fromId = useSelector(
-    (state: RootState) => state.portalMap.inspectPortalId
+    (state: RootState) => state.portalMap.inspectFromId
   )
+  const toId = useSelector((state: RootState) => state.portalMap.inspectToId)
+
+  const size = useSelector((state: RootState) => state.portalMap.size)
+
+  const timeLeft = useSelector((state: RootState) => state.portalMap.timeLeft)
+
```

Comment:
You can and should do this as a single statement. That way you're not looking up state so much and since they are all coming from the same object.

```suggestion
  const { toId, size, timeLeft } = useSelector((state: RootState) => state.portalMap)

```